doc/source/whatsnew/v0.25.0.rst

```diff
@@ -266,7 +266,7 @@ Datetimelike
 ^^^^^^^^^^^^
 
 - Bug in :func:`to_datetime` which would raise an (incorrect) ``ValueError`` when called with a date far into the future and the ``format`` argument specified instead of raising ``OutOfBoundsDatetime`` (:issue:`23830`)
--
+- Bug in :func:`to_datetime` which would raise ``InvalidIndexError`` when called with ``cache=True``, with ``arg`` including several different "not a numbers" (:issue:`22305`)
```

Comment:
show more text of the error: InvalidIndexError: Reindexing only valid with uniquely valued Index objects

this is also confusing what you wrote: ``including several different "not a numbers"``

pls re-word